dbt/models/iasworld/schema/iasworld.dweldat.yml

```diff
@@ -1126,24 +1165,19 @@ sources:
                   meta:
                     description: >
                       user20 (No. of Commercial Units) should not be null
-          - name: user24
-            description: '{{ doc("shared_column_card_proration_rate") }}'
-            tests:
-              - not_null:
-                  name: iasworld_dweldat_card_proration_rate_not_null
+              - accepted_values:
+                  name: iasworld_dweldat_char_ncu_between_0_and_5
+                  values: ['0', '1', '2', '3', '4', '5']
                   additional_select_columns: *select-columns
                   config: *unique-conditions
                   meta:
-                    description: user24 (Proration %) should not be null
+                    description: >
+                      user20 (No. of Commercial Units) should be an integer between 0 and 5
+          - name: user24
+            description: '{{ doc("shared_column_card_proration_rate") }}'
           - name: user30
             description: '{{ doc("shared_column_char_porch") }}'
             tests:
-              - not_null:
-                  name: iasworld_dweldat_char_porch_not_null
```

Comment:
Apparently, `char_porch` (like a few other chars that follow) can be null if the dwelling does not have a porch. I'm not sure if I should be updating the docs in cases like these; happy to do so if you think it's useful to have information in the column docs on what nulls mean.

Reply:
I always assumed that but never had it explicitly confirmed. Let's definitely update the docs.